setup.py

```diff
@@ -0,0 +1,62 @@
+from setuptools import setup, find_packages
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='django-ocs-observation-portal',
+    description='The Observatory Control System (OCS) Observation Portal django apps',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    url='https://observatorycontrolsystem.github.io',
+    author='Observatory Control System Project',
+    author_email='ocs@lco.global',
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Science/Research',
+        'License :: OSI Approved :: BSD License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.7',
+        'Topic :: Scientific/Engineering :: Astronomy',
+        'Topic :: Scientific/Engineering :: Physics'
+    ],
+    keywords=['observations', 'astronomy', 'astrophysics', 'cosmology', 'science', 'ocs', 'observatory'],
+    packages=find_packages(),
+    use_scm_version=True,
```

Comment:
Oh cool, so does using `setuptools_scm` mean that the version of the package uploaded to pypi is the git tag?

Reply:
I believe so, but I haven't run the pipeline on a tag yet.